src/sdgio-edit.owl

```diff
@@ -2727,6 +2749,22 @@ AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020235> "Mult
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020235> "improvement of natural resources and ecosystem services system level outcome"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00020235>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020238> (Increased resilience of the poor to climate change and other shocks)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020238> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020238> "2019-04-28T12:16:49Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020238> "1"^^xsd:integer)
```

Comment:
"1" is not a valid synonym. I think we should have a DbXref with a CURIE-style ID. 
Something like CGIAR-IDO:1.

@Mttbnchtt: Are there no alternate IDs from the various APIs we were informed about?

Reply:
I added the DbXref CGIAR-IDO:1 as an annotation of the label. 
I was unsure about where to put the DbXref, since here we do not have definitions. When we created the SLOs we put DbXref as annotation of the definition. It seems to me that a label  plays a role that is somehow analogous to the role of a definition when there is no definition. So, I added the DbXref as annotation of the label. Let me know if this is wrong. 

Moreover, I do not know what a CURIE-style ID is. I googled the expression but got no information. I simply copied and pasted the ID that you wrote. If it is important for me to know, could you please refer me to a place where I can find more information?

I added the DbXref as an IRI, as last time. Let me know if this is appropriate in this case too. Thanks very much.

Reply:
> I was unsure about where to put the DbXref, since here we do not have definitions.

In this case, it's a DbXref on a class level. In the SLO case, the xref was for the definition, not the whole class.

> It seems to me that a label plays a role that is somehow analogous to the role of a definition when there is no definition. 

No, the label is just a label. For instances, there sometimes isn't a definition in an ontological sense - the class does that work. 

> So, I added the DbXref as annotation of the label. Let me know if this is wrong.

This should be on the class level, not on the label. Generally, you add APs to other APs when they are solely relevant to them, rather than the whole class.

> Moreover, I do not know what a CURIE-style ID is.

It's an OBO convention and follows the NAMESPACE:ID format, like [ENVO:00001000] or [SDGIO:00000000]. 

This old ENVO issue may be informative: https://github.com/EnvironmentOntology/envo/issues/699
http://owlcollab.github.io/oboformat/doc/obo-syntax.html#2.6

@cmungall do we still have to escape the brackets for these?

> I added the DbXref as an IRI, as last time. Let me know if this is appropriate in this case too

As long as it's resolvable, this should work.

Reply:
I am going to resolve this conversation.
Summary of the issue: the xrefs of the IDOs was invalid.
Solution: I modified the xrefs using the URLs provided by @marieALaporte  in #142.